types/airbnb-prop-types/tsconfig.json

```diff
@@ -11,7 +11,10 @@
         "strictFunctionTypes": true,
         "types": [],
         "noEmit": true,
-        "forceConsistentCasingInFileNames": true
+        "forceConsistentCasingInFileNames": true,
+        "paths": {
+            "react": ["../react/v15/index.d.ts"]
```

Comment:
This should just be in `package.json`; paths are only used as a last resort now that we have real package resolution.